roles/accprovision/tasks/net-operator-controller.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Create directory for acc-provision run
+  file:
+    path: "{{ acc_provision_dir_path }}"
+    state: directory
+    mode: '0755'
+
+- name: Get acc-provision configmap
+  k8s_info:
+    api_version: v1
+    kind: ConfigMap
+    name: acc-provision-config
+    namespace: "{{ lookup('env', 'WATCH_NAMESPACE') }}"
+  register: cmap_data
+
+- name: Collecting config-map content
+  set_fact:
+    cmap_content: "{{ cmap_data['resources'][0]}}"
+
+- name: Write cmap YAML
+  copy:
+    dest: "{{ acc_provision_dir_path }}/cmap.yaml"
+    content: "{{ cmap_content }}"
+
+- name: Collecting config-map spec
+  set_fact:
+    cmap_input: "{{ cmap_data['resources'][0]['data']['spec'] }}"
+
```

Comment:
L8 to L28 are not relevant (there is no config-map that is going to be used)

Reply:
the config-map holds the original structure of acc_provision_input yaml, which will be used to create a new input yaml from the CR if we update any value in the CR. The values in CR will be deep merged with its original content. Although, we can discuss it if we are going to have just calico config and nothing else as per related to acc_provision_input.yaml